ui/src/app/app.component.html

```diff
@@ -0,0 +1,74 @@
+<nav class="navbar navbar-expand-md navbar-dark bg-dark">
+  <a class="nav-link" href="#">Payment Simulator</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse"
+          aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarCollapse">
+    <ul class="navbar-nav mr-auto">
+      <!--<li class="nav-item">
+        <a class="nav-link" routerLink="/example" routerLinkActive="active">Example</a>
+      </li>-->
+    </ul>
+  </div>
+</nav>
+<div class="container">
+  <br>
+  <br>
+  <br>
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="form-group landingLabels">
+          <label><b>Merchant</b></label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-group">
+          <label>PAYGATE - TEST SYSTEM</label>
+        </div>
+      </div>
+      <div class="w-100"></div>
+      <div class="col">
+        <div class="form-group landingLabels">
+          <label><b>Reference</b></label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-group">
+          <label>Paygate Test</label>
+        </div>
+      </div>
+      <div class="w-100"></div>
+      <div class="col">
+        <div class="form-group landingLabels">
+          <label><b>Transaction date</b></label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-group">
+          <label>date</label>
+        </div>
+      </div>
+      <div class="w-100"></div>
+      <div class="col">
+        <div class="form-group landingLabels">
+          <label><b>Amount</b></label>
+        </div>
+      </div>
+      <div class="col">
+        <div class="form-group">
+          <label>R 00.00 (ZAR)</label>
+        </div>
+      </div>
+    </div>
+  </div>
+  <hr>
+  <router-outlet></router-outlet>
+  <hr>
+  <div class="landingfooterOptions">
+    <p>Powered by Paygate</p>
```

Comment:
I'm not sure what my thoughts are here... We probably don't want people to think that this tool is created by Paygate... But we do somehow want to indicate that we are simulating paygate...

Reply:
@trojanc , I mocked the screens from the document. I could have just taken screenshots but saw it was just as quick to code it quick and dirty as we said in the Alt room. Again, would have been resolved if there was a design session